src/components/object-info/ObjectInfo.jsx

```diff
@@ -25,7 +36,8 @@ const nodeStyles = {
   'eth-block-list': { shortName: 'ETH', name: 'Ethereum Block List', color: '#383838' },
   'eth-tx-trie': { shortName: 'ETH', name: 'Ethereum Tx Trie', color: '#383838' },
   'eth-tx': { shortName: 'ETH', name: 'Ethereum Tx', color: '#383838' },
-  'eth-state-trie': { shortName: 'ETH', name: 'Ethereum State Trie', color: '#383838' }
+  'eth-state-trie': { shortName: 'ETH', name: 'Ethereum State Trie', color: '#383838' },
+  'hamt-sharded-directory': { shortName: 'HAMT', name: 'HAMT Sharded Directory', color: '#9f28ca'}
```

Comment:
nit: let's use the same colors ad PB, this is effectively a special version, so maybe:

```suggestion
  'hamt-sharded-directory': { shortName: 'PB+H', name: 'HAMT-Sharded dag-pb Directory', color: '#244e66'}
```

This way it will look good when traversing a dag and having sharded and not sharded dag-pb's, plus easier on the eyes on the main page;

>  ![2023-05-26_13-39](https://github.com/ipfs/ipld-explorer-components/assets/157609/27e48979-74b0-476a-8a2a-cf4f29325a0b)